new-packaging/README.md

```diff
@@ -0,0 +1,21 @@
+You need `xmlstarlet`, `dos2unix`
+
+From `/path/to/neo4j` root directory run `mvn install`
+
+`mvn clean install -DskipTests`
+
+you need `cypher-shell` packaged
+
+`make out/cypher-shell.zip`
+
+and then,
+
+`cp out/cypher-shell.zip /path/to/neo4j`
+
+From `/path/to/neo4j/packaging directory` run `mvn install`
```

Comment:
This step is not required if you just move the cypher-shell instructions first and then tell people to build neo4j.